pkg/command/domain/human_readable_flags_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright © 2019 The Knative Authors
+//
```

Comment:
Same as above

Reply:
agree, code changed. Thank you.

Reply:
Same as above.

Reply:
Hi @zhanggbj , Not sure what happens. I already made the change of 2019 --> 2020. You can check "Files changed" tab. Thank you.